internal/pkg/describe/worker_service.go

```diff
@@ -17,25 +17,21 @@ import (
 
 // WorkerServiceDescriber retrieves information about a worker service.
 type WorkerServiceDescriber struct {
-	app             string
-	svc             string
-	enableResources bool
-
-	store                DeployedEnvServicesLister
-	svcDescriber         map[string]ecsSvcDescriber
-	initServiceDescriber func(string) error
+	*ecsServiceDescriber
 }
 
 // NewWorkerServiceDescriber instantiates a worker service describer.
 func NewWorkerServiceDescriber(opt NewServiceConfig) (*WorkerServiceDescriber, error) {
 	describer := &WorkerServiceDescriber{
-		app:             opt.App,
-		svc:             opt.Svc,
-		enableResources: opt.EnableResources,
-		store:           opt.DeployStore,
-		svcDescriber:    make(map[string]ecsSvcDescriber),
+		ecsServiceDescriber: &ecsServiceDescriber{
+			app:             opt.App,
+			svc:             opt.Svc,
+			enableResources: opt.EnableResources,
+			store:           opt.DeployStore,
+			svcDescriber:    make(map[string]ecsSvcDescriber),
```

Comment:
Can we rename the interface `ecsSvcDescriber` to `ecsStack` it's a bit surprising to have both a struct named `ecsServiceDescriber` and an interface named `ecsSvcDescriber` 😃